space2stats_api/src/space2stats/handler.py

```diff
@@ -0,0 +1,22 @@
+"""space2stats lambda handler."""
```

Comment:
@vincentsarago; do you see any potential issues with deployment? Should we try deploying before merging?

Reply:
I think it will be a good idea 👍

Reply:
Ran into an issue with deployment. Will debug today...